### PR TITLE
Add information about Git Rev News

### PR DESCRIPTION
**Pull request by chriscool**
Add a "News Letter" section with information about Git Rev News.

Closes https://github.com/git/git-scm.com/issues/1476

cc @phil-blain @pedrorijo91

### CODE REVIEW

**Comment by phil-blain** [comment]
Thanks @chriscool ! looks good. I'd update the PR description :
`s/See also:/Closes`

so that the issue is automatically closed upon merging this one :)

**Comment by chriscool** [comment]
@phil-blain done, thanks!

File: app/views/community/index.html.erb [hunk 1]
```diff
@@ -79,6 +79,12 @@
     If you need specific help about one of the for-profit Git hosting sites, you might try their own IRC channels (such as <span class="highlight fixed">#github</span> or <span class="highlight fixed">#gitlab</span>) on the same IRC server.
   </p>
 
+  <h2> News Letter </h2>
+
+  <p>
+    There is a monthly community news letter called <a href="https://git.github.io/rev_news/rev_news/">"Git Rev News"</a>, with <a href="https://git.github.io/rev_news/archive/">its archive</a> and <a href="https://git.github.io/rev_news/">its latest edition</a>.
+  </p>
```
Comment (pedrorijo91) [inline app/views/community/index.html.erb:86]:
```suggestion
  <p>
    There is a monthly community news letter called <a href="https://git.github.io/rev_news/rev_news/">"Git Rev News"</a>, with <a href="https://git.github.io/rev_news/archive/">its archive</a> and <a href="https://git.github.io/rev_news/">its latest edition</a>. Information on how to subscribe can be found in the <a href="https://git.github.io/rev_news/rev_news/">dedicated website</a>
  </p>
```

File: app/views/community/index.html.erb [hunk 2]
```diff
@@ -79,6 +79,12 @@
     If you need specific help about one of the for-profit Git hosting sites, you might try their own IRC channels (such as <span class="highlight fixed">#github</span> or <span class="highlight fixed">#gitlab</span>) on the same IRC server.
   </p>
 
+  <h2> News Letter </h2>
```
Comment (pedrorijo91) [inline app/views/community/index.html.erb:82]: I'm not 100% sure of this, but I've always seen written Newsletter and not News Letter.

https://www.quora.com/Which-is-the-correct-spelling-newsletter-or-news-letter-Why

Comment (chriscool) [reply app/views/community/index.html.erb]: Thanks I changed it to "Newsletter", squashed everything and force pushed.

**Comment by pedrorijo91** [comment]
looks cool :) https://git-scm-pr-1477.herokuapp.com/community

**Comment by phil-blain** [comment]
A period is missing at the end of the last sentence.

Also (nitpick, you might disagree in this one) I’d write "webpage" instead of "website".

**Comment by chriscool** [comment]
@phil-blain ok thanks! I added a period and also changed "in the dedicated website" to "on the dedicated webpage". I hope everything is fine now!

**Pull request merged**